images/linux/scripts/installers/git.sh

```diff
@@ -8,14 +8,12 @@
 source $HELPER_SCRIPTS/install.sh
 
 ## Install git
-add-apt-repository ppa:git-core/ppa -y
 apt-get update
 apt-get install git -y
```

Comment:
is the version in the repo fresh enough? I think it's pretty outdated at least for ubuntu 16 & 18